spec.emu

```diff
@@ -300,7 +476,8 @@ contributors: Surma, Dan Ehrenberg
       <emu-clause id="sec-hostresolveimportedmodule" type="host-defined abstract operation">
         <h1>
           HostResolveImportedModule (
-            _referencingScriptOrModule_: a Script Record, a Module Record, or *null*,
+            <del>_referencingScriptOrModule_: a Script Record, a Module Record, or *null*,</del>
+            <ins>_resolutionBase_: unknown,</ins>
```

Comment:
I tried looking at the HTML spec to integrate this change, and we still need to pass the whole _referencingScriptOrModule_ Record. This is because HTML stores a reference to the module map (their `(url, moduleType) -> Module Record` cache) in every Module Record's [[HostDefined]] slot, and it needs to access it in HostResolveImportedModule.

I _think_ that it could be stored in the Realm's [[HostDefined]] slot instead, rather than being explicitly passed around between modules, but it looks like a big refactor for a very small gain. @guybedford / @littledan maybe you know _why_ it's stored in every module and not at the Realm level?